client/.eslintrc

```diff
@@ -1,23 +1,29 @@
 {
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
-      "ecmaVersion": 2021,
-      "sourceType": "module"
+    "ecmaVersion": 2021,
+    "sourceType": "module"
   },
   "extends": [
-      "plugin:@typescript-eslint/recommended",
-      "plugin:react/recommended",
-      "plugin:prettier/recommended",
-      "plugin:react-hooks/recommended"
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:prettier/recommended",
+    "plugin:react-hooks/recommended"
   ],
   "settings": {
-    "react": { "version": "detect"}
+    "react": { "version": "detect" }
   },
   "rules": {
     // suppress errors for missing 'import React' in files
-   "react/react-in-jsx-scope": "off",
-   "react/prop-types": "off",
-   "react-hooks/rules-of-hooks": "error",
-   "react-hooks/exhaustive-deps": "error"
+    "react/react-in-jsx-scope": "off",
+    "react/prop-types": "off",
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "error",
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
+      }
+    ]
```

Comment:
Reason for editing our linting file? If the linting isn't working your machine generally not preferred to change it for all the other developers on the team, best to get it working on your machine.

Reply:
My apologies I did not realize it will change the code base for other developers. You are that right linting was not working for me, so I added the script. Sorry for the inconvenience caused for that.